pkg/sfu/downtrack.go

```diff
@@ -244,6 +245,7 @@ func (d *DownTrack) Bind(t webrtc.TrackLocalContext) (webrtc.RTPCodecParameters,
 // because a track has been stopped.
 func (d *DownTrack) Unbind(_ webrtc.TrackLocalContext) error {
 	d.bound.Store(false)
+	d.stopKeyFrameRequester()
```

Comment:
I also added this in `Unbind` just to be sure.

Reply:
We can probably remove this. What do you think?

Reply:
agree to remove

Reply:
Done in https://github.com/livekit/livekit-server/pull/527/commits/b0b7eeb8639917069895eee76386bd131f07e156